pyclean/modern.py

```diff
@@ -55,6 +55,13 @@
         '.ruff_cache/**/*',
         '.ruff_cache/',
     ],
+    'pyright': [
+        '.pyright-app-cache-**/*',
```

Comment:
What's the intention of this glob pattern?

Is it "Any number of subdirectories deep inside folders matching `.pyright-app-cache-*/`"? – If yes, shouldn't the pattern be like that?

```suggestion
        '.pyright-app-cache-*/**/*',
```